model/registry/test.yaml

```diff
@@ -0,0 +1,104 @@
+groups:
+  - id: registry.test.suite
+    prefix: test
```

Comment:
```suggestion
    prefix: test.suite
```

Why not extract it to the prefix and update all attributes to have id without prefix `suite`

Reply:
Sure, I can move `suite` to be the prefix.